public/static/locales/en/form-builder.json

```diff
@@ -363,5 +363,12 @@
   "dyanamicRow": "Repeating question",
   "addToSet": "Add to set",
   "removeFromSet": "Remove from set",
-  "untitledForm": "Unnamed form file"
+  "untitledForm": "Unnamed form file",
+  "questionSet": {
+    "addAnother": {
+      "title": "Button",
+      "placeholder": "Add another",
```

Comment:
Possible to clarify what "another" is here? In French we'll likely need to specify -- is it "add another field/question/button/etc."?

Reply:
Ya I find the english wording here possibly a little off

**Placeholder**
<img width="815" alt="placeholder" src="https://user-images.githubusercontent.com/62242/219662017-a1ad9e61-cf44-4faa-94f0-e3e174974cad.png">

**Input**
<img width="966" alt="Screen Shot 2023-02-17 at 8 11 11 AM" src="https://user-images.githubusercontent.com/62242/219662078-13b2acb4-2ade-434e-b184-527cec16480c.png">

**Output**
<img width="600" alt="Screen Shot 2023-02-17 at 8 08 46 AM" src="https://user-images.githubusercontent.com/62242/219661403-47542a03-0aec-48d2-a682-cc67730b8731.png">